### DISCUSSION TITLE
whisper ctranslate2

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8905] [Ideas]
No text found
err msg please

Date: 10/12/2024 06:38:38
SE: 4.0.8.82 - Microsoft Windows NT 10.0.26100.0 - 64-bit
Message: Calling whisper (CTranslate2) with : C:\Users\ABDULS_X5\AppData\Local\Programs\Python\Python312\Scripts\whisper-ctranslate2.exe --language hi --model "large-v2" --task translate  "C:\Users\ABDULS_X5\AppData\Local\Temp\c07f9efa-0a18-449b-874e-86768fe13af2.wav"
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.

OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
Calling whisper CTranslate2 done in 00:00:04.9573583
Loading result from STDOUT

**Comment by niksedk** [comment]
Please go to the whisper ctranslate forum

**Reply by Policing2024** [reply]
I don't know how to reach the forum pls

**Comment by Policing2024** [comment]
Sir
I like using all the engines.
But ctanslate2 is giving me problem?

**Reply by Policing2024** [reply]
whisper ctranslate2 forum web address please.

**Reply by niksedk** [reply]
https://github.com/SYSTRAN/faster-whisper
https://github.com/Purfview/whisper-standalone-win
